tests/post_training/pipelines/image_classification_timm.py

```diff
@@ -46,19 +46,30 @@ def prepare_model(self) -> None:
         timm_model = replace_timm_custom_modules_with_torch_native(timm_model)
         self.model_cfg = timm_model.default_cfg
         self.input_size = [1] + list(timm_model.default_cfg["input_size"])
```

Comment:
What about the `self.input_size`? Won't we support static models in conformance tests?

Reply:
In proposed PR models exported to ONNX and IR have dynamic 0-axis. I think testing models with dynamic input covers static cases as well.